source/features/hide-zero-packages.tsx

```diff
@@ -0,0 +1,21 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+function init(): void {
+	const packageElem: HTMLElement | null = select(`.numbers-summary a[href='/${getRepoURL()}/packages']`);
```

Comment:
I forgot this suggestion:
```suggestion
	const packagesLabel = select('.numbers-summary [href$="/packages"]');
```